core/tests/Test.Microsoft.Identity.LabInfrastructure/ILabService.cs

```diff
@@ -31,6 +31,6 @@ namespace Test.Microsoft.Identity.LabInfrastructure
 {
     public interface ILabService
     {
-        IEnumerable<IUser> GetUsers(UserQueryParameters query);
+        IUser GetUsers(UserQueryParameters query);
```

Comment:
If you go by returning just one, you should also change the method name to be GetUser